src/generate.ts

```diff
@@ -6,9 +6,11 @@ const getID = (type: string): string => {
   return svgsIndex[type][Math.floor(Math.random() * svgsIndex[type].length)];
 };
 
+type Race = "Caucasian" | "African" | "Asian" | "Latino";
+
 const colors = [
```

Comment:
This would be better as an object, like:

```
const colors = {
    white: {
      skin: ["#f2d6cb", "#ddb7a0"],
      hair: [
        "#272421",
        "#3D2314",
        "#5A3825",
        "#CC9966",
        "#2C1608",
        "#B55239",
        "#e9c67b",
        "#D7BF91",
      ],
    },
    asian: {
      skin: ["#f2e4cb", "#f5dbad"],
      hair: ["#272421", "#0f0902"],
    },
    hispanic: {
      skin: ["#bb876f", "#aa816f", "#a67358"],
      hair: ["#272421", "#1c1008"],
    },
    black: { skin: ["#ad6453", "#74453d", "#5c3937"], hair: ["#272421"] },
};
```

That way you can just glance at it and know immediately what each set of colors corresponds to, rather than having to count the array index and scroll down to see the labels.

I think that would also make the code using `colors` below clearer too.